OpenApiSpecs/exchangeSetService_OpenApi_definition_internal.yaml

```
@@ -0,0 +1,1291 @@
+          description: |
+            Identifies the schema that data adheres to. Incompatible changes to the schema SHOULD be reflected by a different URI. See Versioning of Attributes in the Primer for more information.
+
+            Constraints
+              * If present, MUST be a non-empty URI.
+        subject:
+          type: string
+          description: |
+            This describes the subject of the event in the context of the event producer (identified by source). In publish-subscribe scenarios, a subscriber will typically subscribe to events emitted by a source, but the source identifier alone might not be sufficient as a qualifier for any specific event if the source context has internal sub-structure.
+
+            Identifying the subject of the event in context metadata (opposed to only in the data payload) is particularly helpful in generic subscription filtering scenarios where middleware is unable to interpret the data content. In the above example, the subscriber might only be interested in blobs with names ending with '.jpg' or '.jpeg' and the subject attribute allows for constructing a simple and efficient string-suffix filter for that subset of events.
+
+            Constraints
+              * If present, MUST be a non-empty string
+
+            Example
+              * A subscriber might register interest for when new blobs are created inside a blob-storage container. In this case, the event source identifies the subscription scope (storage container), the type identifies the "blob created" event, and the id uniquely identifies the event instance to distinguish separate occurrences of a same-named blob having been created; the name of the newly created blob is carried in subject.
+              
+                source: https://example.com/storage/tenant/container
+                subject: mynewfile.jpg
+        time:
+          type: string
+          format: date-time
+          description: |
+            Timestamp of when the occurrence happened. If the time of the occurrence cannot be determined then this attribute MAY be set to some other time (such as the current time) by the CloudEvents producer, however all producers for the same source MUST be consistent in this respect. In other words, either they all use the actual time of the occurrence or they all use the same algorithm to determine the value used.
+        data:
+          type: object
+          description: |
+            The event payload. This specification does not place any restriction on the type of this information. It is encoded into a media format which is specified by the datacontenttype attribute (e.g. application/json), and adheres to the dataschema format when those respective attributes are present.
+
+            The `uk.co.admiralty.avcsData.exchangeSetCreated.v1` event payload is described by "#/components/schemas/exchangeSetResponse".
+          allOf:
+            - $ref: "#/components/schemas/exchangeSetResponse"
```

This is for the s100CloudEvent, should be for the s100ExchangeSetResponse

```suggestion
            - $ref: "#/components/schemas/s100ExchangeSetResponse"
```

---

Updated.